public/javascripts/dashboard.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Created by Hemang on 30/06/17.
+ */
+var app = angular.module('Form', [])
+
+app.controller('fc', function ($scope) {
+    $scope.attributes = [
+        {
+            "key" : "Basic Information",
+            "val" : 0
+        },
+        {
+            "key" : "Academic Details",
+            "val" : 1
+        },
+        {
+            "key" : "Experience",
+            "val" : 2
+        },
+        {
+            "key" : "Research and Academic",
+            "val" : 3
+        },
+        {
+            "key" : "Curricular Activities",
+            "val" : 4
+        },
+        {
+            "key" : "Other Declaration",
+            "val" : 5
+        },
+        {
+            "key" : "Uploads",
+            "val" : 6
+        }
+    ]
+    $scope.isSelected = function (value) {
+        return value.val == $scope.selected;
+    }
+    $scope.setSelected = function (value) {
+        $scope.selected = value;
+    }
+
+});
```

Comment:
Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `cli`.*

The issue can be fixed by applying the following patch:

```diff
--- a/public/javascripts/dashboard.js
+++ b/public/javascripts/dashboard.js
@@ -41,4 +41,4 @@
         $scope.selected = value;
     }
 
-});+});
```